tests/components/grid.test.ts

```diff
@@ -662,6 +668,19 @@ describe("Grid component", () => {
       await simulateClick(".o-filter-icon");
       expect(fixture.querySelectorAll(".o-filter-menu")).toHaveLength(1);
     });
+
+    test("Hitting esc key correctly closes the filter menu", async () => {
```

Comment:
can be moved to file `filter_menu.test.ts`
That's probably the first place we would go looking to find if this behavior is tested